src/Manager.cpp

```diff
@@ -11,23 +11,59 @@ Manager::Manager()
 Manager::~Manager()
 {}
 
+/*
+* Adds new node to UniqueTable
+* @param new node label
+* @return id of new node
+*/
 BDD_ID Manager::createVar(const std::string &label)
-{}
+{
+    BDD_ID newVarId = uniqueTable.size();
+    BDD_ID i;
+    for(i = 0; i < newVarId; ++i)          // to check if we already have that variable
```

Comment:
I think for aesthetics reasons you can put the definition for "i" inside the for. (ex for(BDD_ID i = 0; .....)

Reply:
Yes. But I am not sure if definition in for() works for all compilers.

Reply:
This can be leaved like it is then